enable/tests/qt4/test_mouse_wheel.py

```diff
@@ -92,18 +100,29 @@ def test_horizontal_mouse_wheel(self):
                 QtCore.Qt.NoModifier,
                 QtCore.Qt.Horizontal,
             )
-        else:
+        elif is_qt5:
             qt_event = QtGui.QWheelEvent(
                 QtCore.QPoint(0, 0),
                 self.window.control.mapToGlobal(QtCore.QPoint(0, 0)),
                 QtCore.QPoint(200, 0),
                 QtCore.QPoint(200, 0),
                 200,
-                QtCore.Qt.Vertical,
+                QtCore.Qt.Horizontal,
                 QtCore.Qt.NoButton,
                 QtCore.Qt.NoModifier,
                 QtCore.Qt.ScrollUpdate,
             )
+        else:
+            qt_event = QtGui.QWheelEvent(
+                QtCore.QPointF(0, 0),  #pos
+                self.window.control.mapToGlobal(QtCore.QPointF(0, 0)),  # global pos
+                QtCore.QPoint(200, 0),  # pixel delta
+                QtCore.QPoint(200, 0),  # angle delta
+                QtCore.Qt.NoButton,  # mouse buttons
+                QtCore.Qt.NoModifier,  # keyboard modifiers
+                QtCore.Qt.ScrollUpdate,  # phase
+                False,  #inverted
+            )
```

Comment:
maybe move these comments up to start at line 72

Reply:
also maybe would be nice to have similar comments for the qt5 case

Reply:
Actually, the right way improve readability is probably to use keyword arguments.